src/main/java/seedu/address/ui/MainWindow.java

```diff
@@ -180,19 +180,25 @@ public EventListPanel getEventListPanel() {
 
     public void setPersonTab(CommandDetailChange commandDetailChange) {
         if (commandDetailChange == CommandDetailChange.DETAILED) {
+            logger.info("Setting detailed view for Persons");
             personListPanel.setDetailedView();
         } else if (commandDetailChange == CommandDetailChange.SIMPLIFIED) {
+            logger.info("Setting simplified view for Persons");
             personListPanel.setSimplifiedView();
         }
+        logger.info("Switching to Persons tab");
         tabPanePlaceholder.getSelectionModel().select(0);
     }
 
     public void setEventTab(CommandDetailChange commandDetailChange) {
         if (commandDetailChange == CommandDetailChange.DETAILED) {
+            logger.info("Setting detailed view for Events");
```

Comment:
I like how you have accounted for all the various different cases and made sure that logging was done at appropriate junctures.